cluster/ci/Makefile

```diff
@@ -1,7 +1,9 @@
 LATEST = $(shell curl -q "https://storage.googleapis.com/origin-ci-test/branch-logs/origin/master/builds/.latest" 2>/dev/null)/artifacts/rpms
 REPO ?= $(LATEST)
-INSTANCE_PREFIX ?= $(WHAT)
```

Comment:
Is there a reason we can’t make this an unconditional equality?  Ie remove the “?”

Reply:
?? That's what this PR does ??

Reply:
@smarterclayton Any chance getting some clarification on this comment?

Reply:
@smarterclayton This is needed for me to automate pruning, ptal and merge